src/Controller/OcrController.php

```diff
@@ -165,6 +175,15 @@ public function homeAction(): Response {
 		static::$params['available_langs'] = $this->engine->getValidLangs();
 		sort( static::$params['available_langs'] );
 
+		if ( static::$params['engine'] === 'transkribus' ) {
+			// Pre-supply the available line ids for autocompletion in the form.
+			static::$params['available_line_ids'] = $this->engine->getValidLineIds( true, false );
```

Comment:
This array key also needs to be set in `OcrController::$params`, otherwise it throws an error when loading without `?engine=transkribus`:

> Variable "available_line_ids" does not exist.
> in templates/_transkribus_options.html.twig (line 12)